### PR TITLE
Make results reproducitible by setting PyTorch RNG's seed

### LINKED ISSUE #6: Reproducibility of results
Initiate and save state of pseudo random number generators to recreate experiments

### PR DESCRIPTION
**Pull request by Domnyk**
Closes #6

### CODE REVIEW

**Comment by Domnyk** [comment]
TODO: I'll probably use numpy so I can already set numpy RNG's seed

File: src/training/train.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import torchvision.datasets
+from pathlib import Path
+
+def run():
+    imagenet_data_path = Path('../../data/imagenet/')
+    imagenet_data = torchvision.datasets.ImageNet(imagenet_data_path, download=True)
+
+    return imagenet_data
```
[the remark this answers is not in the document]
Comment (Domnyk) [reply src/training/train.py]: I have moved this file to new package called `classifier`.

**Comment by Domnyk** [comment]
numpy RNG's seed is set in 6556b0a.

**Comment by Domnyk** [comment]
This PR introduces fixed state of random number generators. It also introduces a lot other things which isn't the best thing but I don't have will to refactor it
